STEERING-COMMITTEE.md

```diff
@@ -1,27 +1,108 @@
+    1.  The top three contributing companies to Istio are eligible for
+    Contributor Seats, proportional to their contribution.
+        1.  Each company is allocated one seat;
+        1.  The remaining six seats are allocated based on percentage project
+        contribution, with no Contributing Company exceeding 5 seats in total
+        as outlined in section 6.
+1.  There shall be four **Community Seats.**
+    1.  Two **Community Seats** will be elected by the Istio contributors and
+    community, beginning in July 2020.
+        1.  Any [project member](ROLES.md#member) can self-nominate for the
+        election, or nominate another project member with their consent.
+        1.  Elections use time-limited, Condorcet voting.
+        1.  The following are eligible to vote for Community Seats:
+            1.  Project [Members](ROLES.md#member) who have had a merged PR in
+            the 12 months prior to the election; and
+            1.  People who have submitted a voting exception form to the
+            Steering Committee, demonstrating contribution to the Istio project
+            that is of a non-code nature in the 12 months prior to the election,
+            and are granted a vote for the election by a simple majority vote of
+            the Steering Committee.
+        1.  Community Seats are maintained for the term by the individual, even
+        if they change their company affiliation.
+            1.  If an individual changes company affiliation mid-term in a way
+            that is incompatible with the company representation policies in
+            this Charter, the individual may keep their seat for the duration of
+            the term.
+    1.  Two Community Seats will be appointed through a vote of the Steering
+    Committee after the community election.
+        1.  These seats are intended to diversify perspectives and expertise on
+        the Committee.
+    1.  Because the goal of Community Seats is to increase the perspectives on
+    the Steering Committee, employees of a Contributing Company are ineligible
+    to be elected to hold a Community Seat.
+1.  At all meetings of the Steering Committee, at least 60% of the Seats shall
+constitute a quorum for voting purposes. The vote of an affirmative vote of 60%
+of the Seats present at the time of the vote shall be the decision of the
+Steering Committee. However, any changes to the Steering Committee Charter shall
+require both a quorum and a vote of 80% of the Steering Committee Seats.
 
 ## Committee meetings
 
-The Istio Steering Committee meets weekly; currently on Tuesday mornings. 
-Given the nature of the discussions in Steering, meetings are not currently open to the public. Questions around governance are listed as [issues in the community repo](https://github.com/istio/community/labels/steering-governance), and we invite your feedback there. 
+The Istio Steering Committee meets weekly; currently on Tuesday mornings.
```

Comment:
Clarification...the meeting moved to Friday. I would remove the actual day of the week from the charter.

Reply:
Good call. Removing.